satellite/console/consoleweb/server.go

```diff
@@ -112,25 +114,31 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 		server.config.ExternalAddress = "http://" + server.listener.Addr().String() + "/"
 	}
 
-	mux := http.NewServeMux()
+	router := mux.NewRouter()
 	fs := http.FileServer(http.Dir(server.config.StaticDir))
 
-	mux.Handle("/api/graphql/v0", http.HandlerFunc(server.grapqlHandler))
-	mux.Handle("/api/v0/token", http.HandlerFunc(server.tokenHandler))
+	paymentController := consoleapi.NewPayments(logger, service)
+	paymentsRouter := router.PathPrefix("/api/v0/payments").Subrouter()
+	paymentsRouter.Handle("/cards", http.HandlerFunc(paymentController.AddCreditCard)).Methods(http.MethodPost)
```

Comment:
Ah, just noticed this... mux.Router has `HandleFunc`, so you don't have to cast everything to `HandlerFunc`